docs/beginner/tutorial1-window/README.md

```diff
@@ -20,6 +20,11 @@ pollster = "0.2"
 ## Using Rust's new resolver
 As of version 0.10, wgpu require's cargo's [newest feature resolver](https://doc.rust-lang.org/cargo/reference/resolver.html#feature-resolver-version-2). As a result you must include `resolver = "2"` in either the `[package]` section of `Cargo.toml` if you are working on a single crate, or the `[workspace]` section of the root `Cargo.toml` in a workspace.
 
+## env_logger
+It is very important to enable logging via `env_logger::init();`.
+When wgpu hits any error it panics with a useless message, while logging the real error via the log crate.
```

Comment:
The wording here is a bit aggressive. I would use something like "generic" instead of "useless".